src/sfizz/Region.cpp

```diff
@@ -215,6 +215,7 @@ bool sfz::Region::parseOpcode(const Opcode& rawOpcode)
         setRangeStartFromOpcode(opcode, keyRange, Default::keyRange);
         setRangeEndFromOpcode(opcode, keyRange, Default::keyRange);
         setValueFromOpcode(opcode, pitchKeycenter, Default::keyRange);
+        pitchKeycenterFromSample = false;
```

Comment:
So if I follow what would be the most useful would be to have
```c++
if (!pitchKeycenterFromSample)
    setValueFromOpcode(opcode, pitchKeycenter, Default::keyRange);
```
This would mean that to override `pitch_keycenter=sample` you need to explicitely have a `pitch_keycenter=...` opcode. Having `key=...` would just set the low and high key.

Reply:
Bump on this one. I think the above solution is the sensible way to go for now.

Reply:
Like this? c5a1d04